WordPress/gutenberg
# Relocates Font Face and Fonts Library PHP files into Core's fonts directory

## What?


Relocates the PHP files for Font Face and Fonts Library into `lib/experimental/fonts/`.

## Why?


To align the directory structure with Core.

Font Face and Fonts Library are both part of fonts management. WordPress Core has  an existing `wp-includes/fonts/` directory. The `lib/experimental/fonts/` directory was intended to align to Core's `fonts` directory.

## How?


* Font Face: moves its files into `fonts/font-face/`.
* Fonts Library: moves its directory into `fonts`.

## Testing Instructions

All CI jobs should pass ✅ 

Locally running the following command should run all Font Face and Fonts Library tests:

```
npm run test:unit:php:base -- --group fonts
```
